DataFormats/Detectors/CPV/src/Cluster.cxx

```diff
@@ -0,0 +1,61 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "DataFormatsCPV/Cluster.h"
+#include "CPVBase/Geometry.h"
+#include "CPVBase/CPVSimParams.h"
+
+using namespace o2::cpv;
+
+ClassImp(Cluster);
+
+//____________________________________________________________________________
+bool Cluster::operator<(const Cluster& other) const
+{
+  // Compares two Clusters according to their position in the CPV modules
+
+  char cpvmod1 = getModule();
+  char cpvmod2 = other.getModule();
+  if (cpvmod1 != cpvmod2) {
+    return cpvmod1 < cpvmod2;
+  }
+
+  float posX, posZ;
+  getLocalPosition(posX, posZ);
+  float posOtherX, posOtherZ;
+  other.getLocalPosition(posOtherX, posOtherZ);
+  int rowdifX = (int)std::ceil(posX / o2::cpv::CPVSimParams::Instance().mSortingDelta) -
```

Comment:
could you check the consistency of mSortingDelta definition and usage? Its usage in rowdifX calculation (division, which I asked to avoid) here is orthogonal to usage (multiplication, as it should be) on L55.